lib/common/busypoll.c

```diff
@@ -0,0 +1,495 @@
+#include "h2o.h"
+
+#if defined(__linux__) && defined(SO_REUSEPORT) && defined(H2O_HAS_YNL_H)
+
```

Comment:
```suggestion
```

Assuming we do above, we would no longer need to have these ifs.

Reply:
Thank you for the feedback, I've made the suggested changes in dddeb5a219345615ce8495b3280aec6a5de22201